model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -1094,6 +1094,13 @@ message StandardCoders {
   }
 }
 
+message LogicalTypes {
```

Comment:
Yeah I'd be in favor of that. A couple of other suggestions:
- I think it would make sense to move this enum to schema.proto with all the other representations for the type system
- This should document the representation type, argument type, and the meaning of the logical type, as we do in the comment you linked.

Reply:
Done. Thanks for the comment!